### ISSUE TITLE
Rule #920350 Host header is a numeric IP address

### ISSUE THREAD

**Issue by rahulthackkar** [coreruleset/coreruleset#3673]
### Description
It might not be the False Positive, But I have Three CIDR Ranges, I should allow for this rule, can you help me to write a exclusion rule for this?

### How to reproduce the misbehavior (-> curl call)
Set AWS ALB (Load Balancer), and all requests are coming via ALB

### Your Environment

* CRS version (e.g., v3.3.4):v3.3.2
* Paranoia level setting (e.g. PL1) : PL1
* ModSecurity version (e.g., 2.9.6): 2.9.5
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Apache 2.4.52
* Operating System and version: Ubuntu 22.04

### Confirmation

- [x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by Xhoenix** [comment]
Hi @rahulthackkar,

You can use an exclusion rule like this:-

```
SecRule REQUEST_FILENAME "@endsWith /<your-url-path>" \
    "id:1111,\
    phase:1,\
    pass,\
    t:none,\
    nolog,\
    ctl:ruleRemoveTargetById=920350;REQUEST_HEADERS:Host"
```

**Comment by rahulthackkar** [comment]
> Hi @rahulthackkar,
> 
> You can use an exclusion rule like this:-
> 
> ```
> SecRule REQUEST_FILENAME "@endsWith /<your-url-path>" \
>     "id:1111,\
>     phase:1,\
>     pass,\
>     t:none,\
>     nolog,\
>     ctl:ruleRemoveTargetById=920350;REQUEST_HEADERS:Host"
> ```

Hello @Xhoenix Thanks for reply.

I have added below rules in /etc/modsecurity/crs/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf

```
SecRule REMOTE_ADDR "@ipMatch 172.31.32.0/20" \
    "id:1001, \
    phase:1, \
    pass, \
    nolog, \
    ctl:ruleRemoveById=920350"

SecRule REMOTE_ADDR "@ipMatch 172.31.16.0/20" \
    "id:1002, \
    phase:1, \
    pass, \
    nolog, \
    ctl:ruleRemoveById=920350"

SecRule REMOTE_ADDR "@ipMatch 172.31.0.0/20" \
    "id:1003, \
    phase:1, \
    pass, \
    nolog, \
    ctl:ruleRemoveById=920350"
```
Is this a standard  way to write exclusions?

**Comment by sanvu88** [comment]
You can exclude multiple IPs in a single rule

`SecRule REMOTE_ADDR "@ipMatch 172.31.32.0/20 172.31.16.0/20 172.31.0.0/20" \`

**Comment by rahulthackkar** [comment]
> /etc/modsecurity/crs/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf

Yes, Thanks.

Is this a right place for such rules?
/etc/modsecurity/crs/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf

**Comment by azurit** [comment]
> Is this a right place for such rules? /etc/modsecurity/crs/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf

@rahulthackkar Yes.
